pythran/pythonic/types/bool.hpp

```diff
@@ -3,4 +3,35 @@
 
 #include "pythonic/include/types/bool.hpp"
 
+#ifdef ENABLE_PYTHON_MODULE
+
+#include "pythonic/python/core.hpp"
+
+namespace pythonic
+{
+  template <>
+  struct to_python<bool> {
+    static PyObject *convert(bool b)
+    {
+      if (b)
+        Py_RETURN_TRUE;
+      else
+        Py_RETURN_FALSE;
+    }
+  };
+  template <>
+  struct from_python<bool> {
+    static bool is_convertible(PyObject *obj)
+    {
+      return PyBool_Check(obj);
+    }
+    static bool convert(PyObject *obj)
+    {
+      return PyInt_AsLong(obj);
+    }
+  };
+}
+
+#endif
+
```

Comment:
On Mon, Oct 05, 2015 at 12:54:00PM -0700, pbrunet wrote:
> > +  };
> > +  template <>
> > +  struct from_python<bool> {
> > +    static bool is_convertible(PyObject *obj)
> > +    {
> > +      return PyBool_Check(obj);
> > +    }
> > +    static bool convert(PyObject *obj)
> > +    {
> > +      return PyInt_AsLong(obj);
> > +    }
> > +  };
> > +}
> > +
> > +#endif
> > +
> 
> Like for other, can you declare it in include files?

The interface is not exported to other files (one always use the
to_python function instead) so... no?